tests/test_project.py

```diff
@@ -563,5 +572,36 @@ def helper_test_list_strain_maps(project):
             pytest.fail(str(e.str_itr()))
 
 
+def helper_test_add_project(project):
+    """Test add_project API"""
+
+    try:
+        project.add_project("", "", "")
+        pytest.fail("No exception raised when using an invalid parameter")
+    except SherlockAddProjectError as e:
+        assert str(e) == "Add project error: Project name cannot be blank"
+
+    if project._is_connection_up():
+        try:
+            project.add_project("Tutorial Project", "", "")
+            pytest.fail("No exception raised when creating a duplicate test")
```

Comment:
I'd prefer to say, "duplicate project" rather than "duplicate test". It's ambiguous - the code isn't trying to create a test.